iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerTestUtils.java

```diff
@@ -51,6 +51,18 @@ public class HiveIcebergStorageHandlerTestUtils {
           optional(3, "Last_name", Types.StringType.get())
   );
 
+  static final Schema SOURCE_SCHEMA = new Schema(
```

Comment:
why do we need both `SOURCE_SCHEMA` & `TARGET_SCHEMA` they are the same? can't we create some generic object like `USER_CLICKS_SCHEMA`

Reply:
updated.